### PR TITLE
Fix blank user name in slack message

### PR DESCRIPTION
**Pull request by abhinav098**
The issue was, twilio job was getting send before user name was updated. 
So moved it below so that user name is there. Also in the twilio service, it was written as User.current_user.full_name, but it wont be available in the service. So used `@user`.
Please review 
Thanks

### CODE REVIEW

**Comment by namrataukirde** [comment]
@abhinav098 , In brackets we get mobile number. So will it handle email or phone number in brackets scenario?

**Comment by namrataukirde** [comment]
Means what if user using email id to login?

**Comment by abhinav098** [comment]
No the message was the `send_welcome_message` contains the name of the user.

**Comment by abhinav098** [comment]
ohh i see.. sorry to misundertand.

**Comment by abhinav098** [comment]
Bracket should contain the mobile/email for the user which has been used to sign-up. That's what you're saying right?

**Comment by namrataukirde** [comment]
Yes @abhinav098 👍

**Comment by namrataukirde** [comment]
e.g considering mobile number : `SlackSMS (+85291111111) Single-use pin is 3217. GoodCity.HK welcomes you! Enjoy browsing quality goods.(If you didn't request this message, please ignore)`

File: app/services/twilio_service.rb [hunk 1]
```diff
@@ -30,10 +30,11 @@ def send_unread_message_reminder(url)
   # options[:to] = "+85261111111"
   # options[:body] = "SMS body"
   def send_sms(options)
-    options = {to: @user.mobile}.merge(options)
     if send_to_twilio?
+      options = { to: @user.mobile }.merge(options)
       TwilioJob.perform_later(options)
     else
+      options = { to: @user.mobile.presence || @user.email.presence } .merge(options)
```
Comment (namrataukirde) [inline app/services/twilio_service.rb]: It should be based on what user is using for login and not based on what is present.

Thoughts @abhinav098 ?

Comment (abhinav098) [reply app/services/twilio_service.rb]: Yeah i think you're right.

Comment (namrataukirde) [reply app/services/twilio_service.rb]: 🙂

File: app/services/sendgrid_service.rb [hunk 2]
```diff
@@ -17,6 +17,9 @@ def set_personalizaton_variables
   def send_email
     if send_to_sendgrid?
       sendgrid_instance.client.mail._("send").post(request_body: mail.to_json)
```
Comment (namrataukirde) [inline app/services/sendgrid_service.rb]: We have slack notifications for prod environment as well right now. So I don't think this is good approach to fix this.

**Pull request closed without merging**